### PR TITLE
Support database option group

### PR DESCRIPTION
**Pull request by douglasfraser-aviva**
Attempt to resolve add support for aws_db_option_group #5

### CODE REVIEW

File: modules/db_option_group/outputs.tf [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# DB option group
+output "this_db_option_group_id" {
+  description = "The db option group id"
+  value       = "${aws_db_option_group.this.id}"
```
Comment (antonbabenko) [inline modules/db_option_group/outputs.tf]: Use this instead: `value       = "${element(split(",", join(",", aws_db_option_group.this.*.id)), 0)}"`

File: modules/db_option_group/outputs.tf [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+}
+
+output "this_db_option_group_arn" {
+  description = "The ARN of the db option group"
+  value       = "${aws_db_option_group.this.arn}"
```
Comment (antonbabenko) [inline modules/db_option_group/outputs.tf]: And this - `value = "${element(split(",", join(",", aws_db_option_group.this.*.arn)), 0)}"`

File: modules/db_option_group/outputs.tf (hunk 1 above)
Comment (zachawilson) [reply modules/db_option_group/outputs.tf]: @antonbabenko  This PR looks useful. Were these changes to output the only thing holding the PR back?

Comment (antonbabenko) [reply modules/db_option_group/outputs.tf]: Yes, I think so but also this PR should be rebased to use the latest version of the module.

**Comment by antonbabenko** [comment]
[v1.17.0](https://github.com/terraform-aws-modules/terraform-aws-rds/releases/v1.17.0) has been released with this feature.

**Pull request closed without merging**